nuxeo-core/nuxeo-core-storage-sql/nuxeo-core-storage-sql-test/src/test/java/org/nuxeo/ecm/core/storage/sql/TestSQLBackendUpgrade.java

```diff
@@ -140,15 +153,15 @@ public void testBasicsUpgrade() throws Exception {
         subjects = nodea.getCollectionProperty("tst:subjects").getStrings();
         tags = nodea.getCollectionProperty("tst:tags").getStrings();
         assertEquals(Arrays.asList("z", "c"), Arrays.asList(subjects));
-        assertEquals(Arrays.asList("3"), Arrays.asList(tags));
+        assertEquals(Collections.singletonList("3"), Arrays.asList(tags));
```

Comment:
I don't see the point of this, especially for tests: it's just more verbose.

Reply:
@kevinleturc

Reply:
It's just because Intellij print me this kind of stuff, agree that this is test, verbose etc... your calls